__tests__/journals.test.js

```diff
@@ -0,0 +1,40 @@
+import pool from '../server/utils/pool.js';
+import setup from '../data/setup.js';
+import request from 'supertest';
+import app from '../server/app.js';
+import UserService from '../server/services/UserService.js';
+
+const mockUser = {
+  firstName: 'momo',
+  lastName: 'cat',
+  email: 'momo@momo.com',
+  password: 'momomomo',
+};
+const mockUser2 = {
+  firstName: 'Stu',
+  lastName: 'Cat',
+  email: 'stu@stu.com',
+  password: 'stuthecat',
+};
+
+const registerAndLogin = async (userProps = {}) => {
+  const password = userProps.password ?? mockUser.password;
+
+  // Create an "agent" that gives us the ability
+  // to store cookies between requests in a test
```

Comment:
I like all the comments on these tests. I've definitely messed up and used multiple agent requests in the past without realizing the cookie was not being passed on.